### PR TITLE
Fix and update the contents of image_cls.

### PR DESCRIPTION
**Pull request by lm-lily**
Add missing lines of configuration setting and change libdlk_x86.so to lib_x86.so in the part of "Run inference script on x86 Linux".

### CODE REVIEW

File: docs/tutorial/image_cls.md [hunk 1]
```diff
@@ -149,9 +162,9 @@ output
  │   └── soc_system.dtb
  ├── models
  │   ├── lib (include trained model library)
- │   │   ├── libdlk_arm.so
- │   │   ├── libdlk_fpga.so
- │   │   └── libdlk_x86.so
+ │   │   ├── lib_arm.so
```
Comment (iizukak) [inline docs/tutorial/image_cls.md]: @lm-lily 
These name like `libdlk_arm.so` are name in current master branch.
Not bug.

File: docs/tutorial/image_cls.md [hunk 2]
```diff
@@ -72,6 +72,13 @@ Below is an example configuration.
   batch size (integer):  64
   image size (integer x integer):  32x32
   how many epochs do you run training (integer):  100
+  select optimizer:  Momentum
```
Comment (iizukak) [inline docs/tutorial/image_cls.md]: Thank you for update 👍

File: docs/tutorial/image_cls.md (hunk 1 above)
Comment (lm-lily) [reply docs/tutorial/image_cls.md]: Thank you @iizukak san, I cannot find libdlk_x86.so, instead I found lib_x86.so in the generated files. I shall ignore this?

Comment (iizukak) [reply docs/tutorial/image_cls.md]: @lm-lily 
Please check
https://github.com/blue-oil/blueoil/pull/724

**Comment by tfujiwar** [comment]
@iizukak OA is still pending.

**Comment by iizukak** [comment]
Ownership Approval

**Pull request merged**